sc/social/like/profiles/default/registry.xml

```diff
@@ -1,4 +1,15 @@
 <?xml version="1.0"?>
 <registry>
   <records interface="sc.social.like.controlpanel.ISocialLikeSettings" />
+
+  <record name="plone.app.tiles">
```

Comment:
no, we need to register the tiles in case collective.cover is installed **after** sc.social.like.